style.css

```diff
@@ -72,12 +71,11 @@ body {
 
 .home-page-section {
   background-size: contain;
-  background-repeat: none;
   display: flex;
   flex-direction: column;
-  width: 304px;
   height: 718px;
   background-image: url(./assets/header-shapes\ mobile.svg);
+  background-repeat: no-repeat;
 }
```

Comment:
### Your Homepage
![position-y](https://user-images.githubusercontent.com/93039359/193280681-0fe3338d-2b19-4deb-a702-f09ce9811353.png)

### Figma Template
![figma](https://user-images.githubusercontent.com/86200356/193257644-8a7debca-1b66-4d8e-971e-6ba6ed6d43aa.png)

- [ ] Kindly give your background position-y to make it a little bit lower than your navbar as you can see in the image to follow the Figma template :blush: as suggested by the previous reviewer :+1:

Reply:
hey
I have done the changes on the desktop part of the CSS, am sorry I feel you highlighted the mobile version.Thank you very much